pkg/gofr/datasource/cassandra/cassandra.go

```diff
@@ -177,6 +176,40 @@ func (c *Client) ExecCAS(dest interface{}, stmt string, values ...interface{}) (
 	return applied, err
 }
 
+func (c *Client) NewBatch(batchType int) error {
+	const (
+		loggedBatch = iota
+		unloggedBatch
+		counterBatch
+	)
+
+	var err error
+
+	switch batchType {
+	case loggedBatch, unloggedBatch, counterBatch:
+		c.cassandra.batch = c.cassandra.session.newBatch(gocql.BatchType(batchType))
+
+	default:
+		err = errUnsupportedBatchType
+	}
+
+	return err
+}
+
+func (c *Client) BatchQuery(stmt string, values ...interface{}) {
```

Comment:
```suggestion
func (c *Client) BatchQuery(stmt string, values ...any) {
```

Reply:
`any` was introduced to provide a simpler way to write `interface{}` in Go 1.18

You can provide some information here

https://stackoverflow.com/questions/71628061/difference-between-any-interface-as-constraint-vs-type-of-argument

I tend to see `interface{}` added to new code to be an archaism, even it's still valid.

That's why I suggested it.

But you can ignore